### PR TITLE
Update frontier_silicon documentation 

### PR DESCRIPTION
**Pull request by wlcrs**
## Proposed change

Update the documentation to reflect the changes in behavior due to the overhaul of the frontier_silicon implementation (cfr. https://github.com/home-assistant/core/pull/64365)

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/64365
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/frontier_silicon.markdown [hunk 1]
```diff
@@ -4,7 +4,7 @@ description: Instructions on how to integrate Frontier Silicon Internet Radios i
 ha_category:
   - Media Player
 ha_iot_class: Local Push
-ha_release: '0.40'
+ha_release: '2022.1'
```
Comment (frenck) [inline source/_integrations/frontier_silicon.markdown]: This version marks the introduction (not the last update).
Please revert this change.

File: source/_integrations/frontier_silicon.markdown [hunk 2]
```diff
@@ -17,49 +17,20 @@ This integration provides support for Internet Radios based on the [Frontier Sil
...
+* Roberts: [Roberts Stream 94i]
 * Some models from: Auna, Technisat, Revo, Pinell, Como Audio
 
-This integration was developed and tested with a Hama [DIR3110] and a Medion [MD 87466].
+This integration was developed and tested with a [Roberts Stream 94i].
 
 ## Configuration
 
-Your Frontier Silicon based device should be automatically discovered by Home Assistant. The auto-discovery service assumes that the device uses the default PIN code: *1234*. If you have changed the PIN code, the auto-discovery will fail as Home Assistant will not be able to connect to the device. You can set the PIN code of your device (depending on manufacturer) under:
+The integration supports automatic discovery of your Internet Radio. If you need to setup the device manually, please provide the device URL, which is typically something like `http://[host]/device`. Some models use a separate port (2244) for API access, this can be verified by visiting http://[host]:[port]/device.
```
Comment (frenck) [inline source/_integrations/frontier_silicon.markdown]:
```suggestion
The integration supports automatic discovery of your Internet Radio. If you need to set up the device manually, please provide the device URL, which is typically something like `http://[host]/device`. Some models use a separate port (2244) for API access, this can be verified by visiting `http://[host]:[port]/device`.
```

File: source/_integrations/frontier_silicon.markdown [hunk 3]
```diff
@@ -17,49 +17,20 @@ This integration provides support for Internet Radios based on the [Frontier Sil
 * Medion: [Medion Radios]
 * Silvercrest: [SIRD 14 C2]
 * Teufel: [Radio 3sixty(2019)]
+* Roberts: [Roberts Stream 94i]
 * Some models from: Auna, Technisat, Revo, Pinell, Como Audio
 
-This integration was developed and tested with a Hama [DIR3110] and a Medion [MD 87466].
+This integration was developed and tested with a [Roberts Stream 94i].
 
 ## Configuration
```
Comment (frenck) [inline source/_integrations/frontier_silicon.markdown]:
```suggestion
## Prerequisites
```

**Review by frenck — CHANGES_REQUESTED** [review]
I miss the Home Assistant configuration part?

File: source/_integrations/frontier_silicon.markdown [hunk 4]
```diff
@@ -8,58 +8,38 @@ ha_release: '0.40'
...
+ha_ssdp: true
+ha_config_flow: true
 ---
 
 This integration provides support for Internet Radios based on the [Frontier Silicon chipset]. Some of the manufacturers which offer products based on these chips include: Hama, Medion, Slivercrest, Auna, Technisat, Revo, Pinnel, etc. These devices will be usually controlled by the [UNDOK] app.
 
+{% include integrations/config_flow.md %}
+
 ## Supported Models
+Frontier Silicon is used by many different brands of radio manufacturers.
```
Comment (frenck) [inline source/_integrations/frontier_silicon.markdown]:
```suggestion
## Supported Models

Frontier Silicon is used by many different brands of radio manufacturers.
```

File: source/_integrations/frontier_silicon.markdown [hunk 5]
```diff
@@ -8,58 +8,38 @@ ha_release: '0.40'
...
   - media_player
+ha_codeowners:
+  - '@wlcrs'
+ha_ssdp: true
+ha_config_flow: true
 ---
 
 This integration provides support for Internet Radios based on the [Frontier Silicon chipset]. Some of the manufacturers which offer products based on these chips include: Hama, Medion, Slivercrest, Auna, Technisat, Revo, Pinnel, etc. These devices will be usually controlled by the [UNDOK] app.
 
+{% include integrations/config_flow.md %}
```
Comment (frenck) [inline source/_integrations/frontier_silicon.markdown]: I think this makes more sense to add after the prerequisites?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/frontier_silicon.markdown (hunk 5 above)
Comment (frenck) [reply source/_integrations/frontier_silicon.markdown]: So maybe you misunderstood me? Thus unresolving.

I think the order should be:

- Supported models
- Prerequiresites
- Configuration

Comment (wlcrs) [reply source/_integrations/frontier_silicon.markdown]: Should be ok now, sorry!

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by frenck — CHANGES_REQUESTED** [review]
@wlcrs A wild merge conflict appeared out of nowhere! 🙈 

Could you take a look?

Thanks! 👍 

../Frenck

File: source/_integrations/frontier_silicon.markdown (hunk 4 above)
Comment (frenck) [reply source/_integrations/frontier_silicon.markdown]: Unresolved, as it was not addressed.

File: source/_integrations/frontier_silicon.markdown [hunk 6]
```diff
@@ -93,15 +72,16 @@ Overview of the info dialog:
 
 ## Development
 
-Support is provided through the Python [fsapi] module. The Python module was developed by using the documentation provided by [flammy] and
+Support is provided through the Python [afsapi] module. The Python module was developed by using the documentation provided by [flammy] and
 is based on [tiwillam]'s fsapi project. Special thanks to both developers, this integration would have not been possible without their work.
```
Comment (frenck) [inline source/_integrations/frontier_silicon.markdown]: I think this section just needs to go. It isn't relevant for the end-user of Home Assistant.

File: source/_integrations/frontier_silicon.markdown [hunk 7]
```diff
@@ -93,15 +72,16 @@ Overview of the info dialog:
...
 
 ## Notes and Limitations
 
-<div class='note warning'>
+<div class='note warning' name="fsapi-session-note">
```
Comment (frenck) [inline source/_integrations/frontier_silicon.markdown]:
```suggestion
<div class='note warning'>
```

File: source/_integrations/frontier_silicon.markdown [hunk 8]
```diff
@@ -11,58 +11,34 @@ ha_platforms:
...
-  required: false
-  default: empty
-  type: string
-{% endconfiguration %}
-
-Some models use a separate port (2244) for API access, this can be verified by visiting http://[host]:[port]/device.
+The default PIN for Frontier Silicon-based devices is 1234. You can set the PIN code of your device (depending on manufacturer) under:
+*MENU button > Main Menu > System setting > Network > NetRemote PIN setup*
 
 In case your device (friendly name) is called *badezimmer*, an example automation can look something like this:
```
Comment (frenck) [inline source/_integrations/frontier_silicon.markdown:46]: Let's move the example us using it, under the config flow procedures.

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @wlcrs 👍 

../Frenck
